packages/core/src/components/ScreenContainer.tsx

```diff
@@ -89,6 +76,10 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
   },
+  scrollViewContainer: {
+    flexGrow: 1,
+    flex: undefined,
```

Comment:
@donald this is a Screen Container.  It wraps all components and covers both safe / unsafe view.  If use a View and nest all children in there, then you should be able to control how much space you eat.

So especially for ScrollView ... it will actually "break" stuff if you pass in a flex: 1.  It needs to always be set to flexGrow: 1